### PR TITLE
magento/devdocs#: Products endpoint

### PR DESCRIPTION
**Pull request by atwixfirster**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will fix content of
- "ProductFilterInput" attributes
- "ProductInterface" attributes

regarding to changes in `https://github.com/magento/magento2/blob/2.3.1/app/code/Magento/CatalogGraphQl/etc/schema.graphqls` (Magento 2 release `version 2.3.1`)

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/graphql/reference/products.html

Thank you!

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/graphql/reference/products.md [hunk 1]
```diff
@@ -173,16 +168,10 @@ Attribute | Data type | Description
...
-`custom_design` | String | A theme that can be applied to the product page
-`custom_design_from` | String | The beginning date when a theme is applied to the product page
-`custom_design_to` | String| The date at which a theme is no longer applied to the product page
-`custom_layout` | String | The name of a custom layout
-`custom_layout_update` | String | XML code that is applied as a layout update to the product page
 `description` | ComplexTextValue | An object that contains detailed information about the product. The object can include simple HTML tags
 `gift_message_available` | String | Indicates whether a gift message is available
 `id` | Int | The ID number assigned to the product
 `image` | ProductImage | An object that contains the URL and label for the main image on the product page
-`is_returnable` | String | Indicates whether the product can be returned. This attribute is defined in the Rma module.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/products.md]: Keep this attribute. Its origin is listed in the description, and I don't want to create a topic that merely defines one or two attributes

File: guides/v2.3/graphql/reference/products.md [hunk 2]
```diff
@@ -191,29 +180,21 @@ Attribute | Data type | Description
...
-`product_links` | [ProductLinks] | An array of [ProductLinks](#ProductLinks) objects
+`product_links` | [ProductLinksInterface] | An array of [ProductLinks](#ProductLinks) objects
 `short_description` | ComplexTextValue | An object that contains a short description of the product. Its use depends on the store's theme. The object can include simple HTML tags
 `sku` | String | A number or code assigned to a product to identify the product, options, price, and manufacturer
 `small_image` | ProductImage | An object that contains the URL and label for the small image used on catalog pages
 `special_from_date` | String | The beginning date that a product has a special price
 `special_price` | Float |  The discounted price of the product
 `special_to_date` | String | The end date that a product has a special price
-`stock_status` | ProductStockStatus | An enumeration describing the stock status of the product. Possible values are `IN_STOCK` and `OUT_OF_STOCK`.
-`swatch_image` | String | The file name of a swatch image. This attribute is defined in the Swatches module.
```
Comment (keharper) [inline guides/v2.3/graphql/reference/products.md]: Keep `swatch_image` and `tax_class_id` for the same reason as above

File: guides/v2.3/graphql/reference/products.md [hunk 3]
```diff
@@ -191,29 +180,21 @@ Attribute | Data type | Description
...
-`tax_class_id` | Int | An ID assigned to a tax class. This attribute is defined in the Tax module.
 `thumbnail` | ProductImage | An object that contains the URL and label for the product's thumbnail image
 `tier_price` | Float | The price when tier pricing is in effect and the items purchased threshold has been reached
 `tier_prices` | [ProductTierPrices] | An array of [ProductTierPrices](#ProductTier) objects
 `type_id` | String | One of `simple`, `virtual`, `bundle`, `downloadable`,`grouped`, `configurable`
 `updated_at` | String | The timestamp indicating when the product was last updated
-`url_key` | String | The part of the URL that identifies the product. This attribute is defined in the `CatalogUrlRewrite` module
```
Comment (keharper) [inline guides/v2.3/graphql/reference/products.md]: Keep `url_key`, `url_path`, and `url_rewrites` for the same reason

**Comment by keharper** [comment]
@atwixfirster I need you to back out a few of the deletions.

File: guides/v2.3/graphql/reference/products.md (hunk 1 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/reference/products.md]: **fixed**

File: guides/v2.3/graphql/reference/products.md (hunk 2 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/reference/products.md]: **fixed**

File: guides/v2.3/graphql/reference/products.md (hunk 3 above)
Comment (atwixfirster) [reply guides/v2.3/graphql/reference/products.md]: **fixed**

**Comment by atwixfirster** [comment]
> @atwixfirster I need you to back out a few of the deletions.

**done**

Many thanks, @keharper !

**Pull request merged**